src/webui/api/torrentscontroller.cpp

```diff
@@ -835,3 +856,91 @@ void TorrentsController::removeCategoriesAction()
     for (const QString &category : categories)
         BitTorrent::Session::instance()->removeCategory(category);
 }
+
+QVariantList TorrentsController::getStickyTrackers(const BitTorrent::TorrentHandle* torrent)
+{
+    static const QString working("Working");
+    static const QString disabled("Disabled");
+    static const QString privateMsg("This torrent is private");
+
+    QVariantList stickyTrackers;
+
+    QVariantMap dht;
+    dht[KEY_TRACKER_URL] = "** [DHT] **";
+    dht[KEY_TRACKER_TIER] = "";
+    dht[KEY_TRACKER_RECEIVED] = "";
+    dht[KEY_TRACKER_MSG] = "";
+    dht[KEY_TRACKER_DOWNLOADED] = "";
+
+    QVariantMap pex;
+    pex[KEY_TRACKER_URL] = "** [PeX] **";
+    pex[KEY_TRACKER_TIER] = "";
+    pex[KEY_TRACKER_RECEIVED] = "";
+    pex[KEY_TRACKER_MSG] = "";
+    pex[KEY_TRACKER_DOWNLOADED] = "";
+
+    QVariantMap lsd;
+    lsd[KEY_TRACKER_URL] = "** [LSD] **";
+    lsd[KEY_TRACKER_TIER] = "";
+    lsd[KEY_TRACKER_RECEIVED] = "";
+    lsd[KEY_TRACKER_MSG] = "";
+    lsd[KEY_TRACKER_DOWNLOADED] = "";
+
+    if (BitTorrent::Session::instance()->isDHTEnabled() && !torrent->isPrivate())
+        dht[KEY_TRACKER_STATUS] = "Working";
```

Comment:
?? you're not using the constant defined above...

Reply:
also I would write it:
```c++
dht[KEY_TRACKER_STATUS] = (BitTorrent::Session::instance()->isDHTEnabled() && !torrent->isPrivate())
    ? "Working"
    : "Disabled";
```

Reply:
Definite goof on my end